pkg/collection/entry/metadata.go

```diff
@@ -21,6 +21,10 @@ func NewMetadata(fileName string) *Metadata {
 	}
 }
 
+func (m *Metadata) SetMimeType(mineType string) {
```

Comment:
IIRC I was already reprimanded for adding this setter to a simple exported type.